src/agent/LibFuzzerDotnetLoader/Program.cs

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Reflection;
+using System.Runtime;
+using System.Runtime.InteropServices;
+using System.Text;
+
+using TestOneSpan = SharpFuzz.ReadOnlySpanAction;
+delegate void TestOneArray(byte[] data);
+
+namespace LibFuzzerDotnetLoader {
+    public class Program {
+        public static void Main(string[] args) {
+            var target = Environment.GetEnvironmentVariable("LIBFUZZER_DOTNET_TARGET");
```

Comment:
We control this fully. I originally had it as 3 env vars, but that was sort of annoying in practice. We could always have it accept either this `:`-delimited string if present, and fall back on individual vars. The latter would mostly be nice for more agility when passing the assembly path.

Reply:
Tracked in #2130.